polaris-react/src/utilities/css.ts

```diff
@@ -26,6 +27,47 @@ export function sanitizeCustomProperties(
   return nonNullValues.length ? Object.fromEntries(nonNullValues) : undefined;
 }
 
+/**
+ * Given an object like so:
+ * {
+ *   sm: 4,
+ *   lg: 6
+ * }
+ * Fill in the blanks starting at >= sm (because it's the first one set):
+ * {
+ *   sm: 4,
+ *   md: 4,
+ *   lg: 6
+ *   xl: 6
+ * }
+ *
+ */
+function makeResponsivePropsContiguous(responsiveProp: {
+  [Breakpoint in BreakpointsAlias]?: string;
+}): {
+  [Breakpoint in BreakpointsAlias]?: string;
+} {
+  const result: {
+    [Breakpoint in BreakpointsAlias]?: string;
+  } = {};
+
+  let prev: string | undefined;
+
+  breakpointAliases.forEach((breakpointAlias) => {
+    if (typeof responsiveProp[breakpointAlias] === 'undefined') {
+      // Don't set values until we've seen at least one
+      if (typeof prev !== 'undefined') {
+        result[breakpointAlias] = prev;
+      }
+    } else {
+      prev = responsiveProp[breakpointAlias];
+      result[breakpointAlias] = prev;
+    }
```

Comment:
```suggestion
    if (responsiveProp[breakpointAlias]) {
      result[breakpointAlias] = responsiveProp[breakpointAlias];
      prev = responsiveProp[breakpointAlias];
    } else if (prev) {
      result[breakpointAlias] = prev;
    }
```

Light suggestion/refactor to improve readability. Test passed locally.

Reply:
Good suggestion 👍 

I'll keep in the `typeof undefined` check because I'm explicitly checking if the key is set to a non-undefined value (ie; falsey values should still be allowed such as `0`)